index.html

```diff
@@ -0,0 +1,125 @@
+        </section>
+        <Span class="divider"><img src="./assets/images/line-diverder.png" alt="section divider"></Span>
+        <section class="skill-stack">
+            <ul class="stack-one">
+                <li><h3>Language</h3></li>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+            <ul class="stack-two">
+                <li><h3>Frameworks</h3></li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>            
+            </ul>
+            <ul class="stack-three">
+                <li><h3>Skill</h3></li>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+        </section> 
+        <Span class="divider"><img src="./assets/images/line-diverder.png" alt="section divider"></Span>
+        <section class="social">
+            <div class="social-media">
+            <ul class="social-media-icon">
+                <li><a href="#"><img src="./assets/icons/github.png" alt="Github"></a></li>
+                <li><a href="#"><img src="./assets/icons/linkedin.png" alt="Linkedin"></a></li>
+                <li><a href="#"><img src="./assets/icons/Vector.png" alt="Angellist"></a></li>
+                <li><a href="#"><img src="./assets/icons/twitter.png" alt="Twitter"></a></li>
+                <li><a href="#"><img src="./assets/icons/medium.png" alt="Medium"></a></li>
+            </ul>
+                <a class="arrow" href="#"><img src="./assets/icons/element-right.png" alt="arrow"></a>   
+            </div>   
+            <div class="experience-title">
+                <h2>MY RECENT <br> WORK</h2>
+            </div> 
+             
+        </section>
+        
+        <Span class="divider"><img src="./assets/images/line-diverder.png" alt="section divider"></Span>
+        <section class="experience-container">
+            <img src="./assets/images/bg.jpg" alt="project image">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li>css<img src="./assets/images/vert-line.png" alt="vertical line"></li>
+                <li>html<img src="./assets/images/vert-line.png" alt="vertical line"></li>    
+                <li>bootstrap<img src="./assets/images/vert-line.png" alt="vertical line"></li>  
+                <li>Ruby</li>
+            </ul>
+
+            <button type="button" class="project-button">See Project</button>
+
```

Comment:
- [ ] Kindly wrap all the projects inside a `ul` tag and `li` tag per project.
_Following the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) is very important for the website's reading and optimization._
**Hint**
```
<section>
  <ul>
    <li>
      // single project's elements here
    </li>
    <li>
      // single project's elements here
    </li>
    <li>
      // single project's elements here
    </li>
  </ul>
</section> // end of the project's section container
```